### ISSUE TITLE
android.presplash_color bug

### ISSUE THREAD

**Issue by pynting** [kivy/buildozer#1487]
Not sure if this is a universal problem, but building my app failed when i uncommented the "android.presplash_color" setting in the buildozer.spec file. 

Relevant log:
```bash
[DEBUG]:        > Task :mergeDebugResources FAILED
[DEBUG]:        /.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/dists/kr/src/main/res/values/strings.xml:3:4: Failed to flatten XML for resource 'presplash_color' with error: Invalid unicode escape sequence in string
[DEBUG]:        "{str}"
[DEBUG]:        /.buildozer/android/platform/build-arm64-v8a_armeabi-v7a/dists/kr/src/main/res/values/strings.xml:3:4: string/presplash_color does not contain a valid string resource.
```

And indeed, in the referenced file:

```xml
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <string name="app_name">Kr</string>
    <string name="private_version">c8a851e76193bfc03b01e371acf67e0f938ddcba</string>
    <string name="presplash_color">'aqua'</string>
    <string name="urlScheme">kivy</string>
</resources>
```
Problem lies in the buildozer file android.py:
```python
        # support for presplash background color
        presplash_color = self.buildozer.config.getdefault('app', 'android.presplash_color', None)
        if presplash_color:
            cmd.append('--presplash-color')
            cmd.append("'{}'".format(presplash_color))
```
Changed 
` cmd.append("'{}'".format(presplash_color))`
into 
` cmd.append("{}".format(presplash_color))`
solved the issue for me

### Versions

* Python: 3.10
* OS: Fedora
* Buildozer: 1.4.1.dev0

**Comment by Olamidipupo-favour** [comment]
I had similar issues.

**Comment by apacelus** [comment]
I made a pr to fix this

**Comment by pynting** [comment]
Perfect! Do I close this issue, or should the administrator(s) do it?

**Comment by apacelus** [comment]
> Perfect! Do I close this issue, or should the administrator(s) do it?

Let them do it, or close it after they accept my pr

**Comment by apacelus** [comment]
@pynting my pr was merged, you can close this issue now

**Comment by misl6** [comment]
Closed via #1497
